README.md

```diff
@@ -30,13 +30,49 @@ Check our research paper ["Can you tell? SSNet - a Sagittal Stratum-inspired Neu
 
 ## Prototype code, see the [licensing agreement](https://github.com/usnistgov/STVM_NLP_Research/blob/master/LICENSE.md) for terms of use.
 
-## Installation Process: 
-The project is written using the [tensorflow](https://www.tensorflow.org/install) library and is compatible with tensorflow 2.x. The dataset used for the experiments can be found at [SLMRD](https://ai.stanford.edu/~amaas/data/sentiment/) which consists of 25K train reviews and 25K test reviews.
+## Installation Process:
+The project is written using the [TensorFlow](https://www.tensorflow.org/install) library and is compatible with TensorFlow 2.x. The dataset used for the experiments can be found at [SLMRD](https://ai.stanford.edu/~amaas/data/sentiment/) which consists of 25K train reviews and 25K test reviews.
+
+The data is downloadable as a `.tar.gz` file which contains text files sorted into positive and negative reviews as well as metadata. For the training and inference code to run properly, `imdb.vocab` should be copied to the `./models` folder (relative to the repository's main directory). If a folder named `train` does not exist in the `models` directory, it will need to be created; then, copy `labeledBow.feat` from the dataset into the `train` folder. Also copy (or move) the `train/neg` and `train/pos` folders from the downloaded data into `train`. This will enable the `model_1...` and `model_2...` scripts to access the training and testing data. Additional preparation may be necessary before training the models or using them to make predictions on the data; see below.
 
 ## WorkFlow of SSNet
 Following is the directory structure:
 <img src="images/directory_structure.png" height="20">
 
+The following is an overview of each file and folder in the repository, intended to reduce challenges with setup for new users. Where applicable, references to the paper and/or instructions for how to run the experiments are included.
```

Comment:
Looks a bit bulky, could you look into ways to make it bit more readable?

Reply:
Yes, I will reformat this (if you think the section is unnecessary I can exclude it, just thought it would be useful to have some more specific info about the project structure since I encountered problems getting everything running when I first tried to run the scripts).